source/Aconcagua-Quantitative-Analysis/UnitOfAccount.class.st

```diff
@@ -0,0 +1,92 @@
+"
+A unit of account is a standard numerical unit of measurement that provides a consistent
+measure of value for goods, services, and assets. It is one of the three primary functions
+of money, alongside being a medium of exchange and a store of value.
+"
+Class {
+	#name : 'UnitOfAccount',
+	#superclass : 'Unit',
+	#instVars : [
+		'symbol',
+		'dimension',
+		'reference'
+	],
+	#category : 'Aconcagua-Quantitative-Analysis',
+	#package : 'Aconcagua-Quantitative-Analysis'
+}
+
+{ #category : 'instance creation' }
+UnitOfAccount class >> on: aReference symbol: unitSymbol [
```

Comment:
I'd rather use `representedBy:` instead of just `symbol:`, and maybe `named:` instead of just `on:`

Reply:
`symbol` usage is already explained in another comment, and `named:` is not a good idea because it is not the name: it's the object used as a reference for this unit of account (it could be a string if you don't care or a more complex object if you care enough)